src/runtime_src/core/include/xgq_cmd_vmr.h

```diff
@@ -116,17 +116,19 @@ enum xgq_cmd_vmr_control_type {
  * log page type
  */
 enum xgq_cmd_log_page_type {
-	XGQ_CMD_LOG_AF_CHECK	= 0x0,
-	XGQ_CMD_LOG_FW		= 0x1,
-	XGQ_CMD_LOG_INFO	= 0x2,
-	XGQ_CMD_LOG_AF_CLEAR	= 0x3,
-	XGQ_CMD_LOG_ENDPOINT	= 0x4,
-	XGQ_CMD_LOG_TASK_STATS  = 0x5,
-	XGQ_CMD_LOG_MEM_STATS	= 0x6,
-	XGQ_CMD_LOG_SYSTEM_DTB	= 0x7,
-	XGQ_CMD_LOG_PLM_LOG	= 0x8,
-	XGQ_CMD_LOG_APU_LOG	= 0x9,
-	XGQ_CMD_LOG_SHELL_INTERFACE_UUID	= 0xa,
+	XGQ_CMD_LOG_AF_CHECK		 = 0x0,
+	XGQ_CMD_LOG_FW			 = 0x1,
+	XGQ_CMD_LOG_INFO		 = 0x2,
+	XGQ_CMD_LOG_AF_CLEAR		 = 0x3,
+	XGQ_CMD_LOG_ENDPOINT		 = 0x4,
+	XGQ_CMD_LOG_TASK_STATS  	 = 0x5,
+	XGQ_CMD_LOG_MEM_STATS		 = 0x6,
+	XGQ_CMD_LOG_SYSTEM_DTB		 = 0x7,
+	XGQ_CMD_LOG_PLM_LOG		 = 0x8,
+	XGQ_CMD_LOG_APU_LOG		 = 0x9,
+	XGQ_CMD_LOG_SHELL_INTERFACE_UUID = 0xa,
+	XGQ_CMD_LOG_DEFAULT_FPT		 = 0xb,
```

Comment:
alignment changes made to keep consistency with vmr common header file

Reply:
look ok as long as the vmr changes made in vmr.